jerry-core/ecma/builtin-objects/ecma-builtin-date-prototype.cpp

```diff
@@ -409,8 +257,30 @@ ecma_builtin_date_prototype_get_utc_milliseconds (ecma_value_t this_arg) /**< th
 static ecma_completion_value_t
 ecma_builtin_date_prototype_get_timezone_offset (ecma_value_t this_arg) /**< this argument */
 {
-  ECMA_BUILTIN_CP_UNIMPLEMENTED (this_arg);
-} /* ecma_builtin_date_prototype_get_timezone_offset */
+  ecma_completion_value_t ret_value = ecma_make_empty_completion_value ();
```

Comment:
I think we can move the time_offset computation into a function. That would clean up things.